Code Syntax Block: Fix button style and adjust its behavior

outdoor2kode opened the pull request:
This PR fixes #99 and increases the number of lines shown based on discussion [here](https://github.com/WordPress/wporg-developer/issues/22#issuecomment-1140675129).

This is how it works currently:
`<= 15 lines, show all` 
`> 15 lines, show expand button`

Note: This PR is based on #100, should be merged after it.

>
> The Copy button is not sticky, but it's also too-sticky.
>
> Scrolling:
![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/767313/172580958-77b99154-18b0-440e-9a23-0608e28a4941.gif)
>
> Scrolling and hiding:
![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/767313/172580978-4d47f715-77c7-474e-af9e-3087fb7b91dc.gif)

Screenshots of Result:
| apply_filters | password_change_email |
| ------------ | ------------------------- | 
| ![Screen Capture on 2022-06-09 at 22-20-36](https://user-images.githubusercontent.com/18050944/172869890-29afd1ab-e7b8-4799-b2f9-41efd59fe6f2.gif) | ![Screen Capture on 2022-06-09 at 12-37-31](https://user-images.githubusercontent.com/18050944/172765249-cb74fe5a-265b-42ce-825d-b287e0b7d2ef.gif) |

I'm not sure if we still want to make the button show on hover as there's a big empty space when it's not hovered?
Personally I like it to always show in the case there, it'd have better UX and it's friendly for mobile device users.

outdoor2kode committed:
Fix button not sticky, and overlapping

outdoor2kode committed:
Fix the scroll bar still shows after expanding

outdoor2kode committed:
Adjust MIN_HEIGHT to show correct amount of lines

outdoor2kode committed:
Increase number of lines shown based on discussion

See https://github.com/WordPress/wporg-developer/issues/22#issuecomment-1140675129.

outdoor2kode commented on source/wp-content/themes/wporg-developer/scss/prism.scss:
```diff
@@ -230,6 +233,6 @@ pre.wp-block-code {
 	&:hover > .wp-code-block-button-container,
 	&:focus-within > .wp-code-block-button-container {
-		display: revert;
+		visibility: visible;
```
Make the display of elements controlled by visibility, so it remains a blank space when not hovering on the code block. Or the UX would be like this:
![Screen Capture on 2022-06-09 at 22-18-05](https://user-images.githubusercontent.com/18050944/172869422-d5247524-42ad-4c1e-a2d0-90b12cd90117.gif)

ryelle committed:
Inject butons before the `pre`, style like a frame header

ryelle commented:
I've rebased the PR since #100 was merged, and the last commit I added styles the buttons a little - they're always visible, and exist outside the pre, styled like a frame around the code.

| With expand button | No expand |
|-----|-----|
| ![Screen Shot 2022-06-09 at 16 50 29](https://user-images.githubusercontent.com/541093/172943502-3181bfb1-37ff-4a1c-b6de-2cce4c7e03cd.png) | ![Screen Shot 2022-06-09 at 16 51 34](https://user-images.githubusercontent.com/541093/172943506-80c0b9ad-a500-44fb-be49-d99b1db2ff93.png) |

Video showing the new scroll behavior:

https://user-images.githubusercontent.com/541093/172943620-88279c70-1513-46f6-bc16-223f316d1e52.mp4

outdoor2kode committed:
Remove unused CSS attributes

outdoor2kode commented:
> I've rebased the PR since https://github.com/WordPress/wporg-developer/pull/100 was merged, and the last commit I added styles the buttons a little - they're always visible, and exist outside the pre, styled like a frame around the code.

Tested on Chrome and Firefox with different screen sizes, everything looks good.

I like how it looks, it's neat and clean!
I've removed three unused CSS attributes and I think it's good to go.
